components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -539,17 +557,23 @@ during post consent handling to get the original ConsentClaimsData object (Assum
                         " for client_id: " + clientId + " of tenantDomain: " + spTenantDomain);
             }
 
+            Map<String, Object> params = new HashMap<>();
+            params.put("clientId", clientId);
+            params.put("prompt", oauth2Params.getPrompt());
             if (hasPromptContainsConsent(oauth2Params)) {
+                OAuth2Util.log("oauth-inbound-service", params, "SUCCESS", null , "hand-over-to-consent-service", null);
                 getSSOConsentService().processConsent(approvedClaimIds, serviceProvider,
                         loggedInUser, value, true);
             } else {
+                OAuth2Util.log("oauth-inbound-service", params, "SUCCESS", null , "hand-over-to-consent-service", null);
                 getSSOConsentService().processConsent(approvedClaimIds, serviceProvider,
                         loggedInUser, value, false);
             }
 
         } catch (OAuthSystemException | SSOConsentServiceException e) {
             String msg = "Error while processing consent of user: " + loggedInUser.toFullQualifiedUsername() + " for " +
                     "client_id: " + clientId + " of tenantDomain: " + spTenantDomain;
+            OAuth2Util.log("oauth-inbound-service", null, "FAILED", "System error occurred." , "process-consent", null);
```

Comment:
Don't we need same for ClaimMetadataException and RequestObjectException

Reply:
+1. added with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1737/commits/eef37f7d62f63b732a61665b40af7c5118f00268